flumine/controls/tradingcontrols.py

```diff
@@ -134,18 +138,21 @@ def _validate(self, order_package):
                     )
                     continue
 
-                # per selection
+            # per selection
+            for (strategy, lookup), orders in package_orders_by_strategy_lookup.items():
+
                 market = self.flumine.markets.markets[order_package.market_id]
```

Comment:
Can you put the market lookup above the loop as it is the same for each strategy so doesn't need to be duplicated.